lib/cell.rb

```diff
@@ -0,0 +1,16 @@
+class Cell
+  attr_reader :coordinate, :ship
+
+  def initialize(coordinate)
+    @coordinate = coordinate
+    @ship = @ship
+  end
+
+  def ship
+  end
+
```

Comment:
I don't think we need this method, since ship is already defined in our initialize method above!

Reply:
Agreed! We do need a #place_ship, and I think that's where I was going with this one.